lenses-cloudwatch-plugin/src/main/scala/io/lenses/alerts/plugin/cloudwatch/CloudWatchAlert.scala

```diff
@@ -0,0 +1,79 @@
+package io.lenses.alerts.plugin.cloudwatch
+
+
+import java.time.format.DateTimeFormatter
+import java.time.ZonedDateTime
+
+import io.lenses.alerting.plugin.Alert
+import io.lenses.alerting.plugin.scalaapi.{Alert => ScalaAlert}
+
+/**
+ * Encapsulates the information for an event in Kafka when something is not running at 100%
+ *
+ * @param labels      - A list of key value pairs. It will contain at least severity.
+ * @param annotations - A list of key value pairs. The keys will contain: summary, source, and docs keys.
+ * @param startsAt    - The time in ISO format of when the alert starts
+ * @param endsAt      - When the alert finishes
+ * @param sourceName  - The source name of the creator of this alert
+ * @param alertId     - A unique identifier for the setting corresponding to this alert
+ */
+case class CloudWatchAlert(labels: Map[String, String],
+                           annotations: Map[String, String],
+                           startsAt: String,
+                           endsAt: Option[String],
+                           sourceName: String,
+                           alertId: Int)
+
+object CloudWatchAlert {
+
+  import io.circe._
+  import io.circe.generic.semiauto._
+
+  implicit val AMAlertEncoder: ObjectEncoder[CloudWatchAlert] = deriveEncoder[CloudWatchAlert]
+
+  implicit class AlertExtension(val alert: Alert) extends AnyVal {
+    def toCWAlert: CloudWatchAlert = {
+      val docs = if (alert.docs.isPresent) Some(alert.docs.get) else Option.empty[String]
```

Comment:
The scala-java8-compat lib has a helper for this: 

https://github.com/scala/scala-java8-compat#converters-between-scalaoption-and-javautil-classes-optional-optionaldouble-optionalint-and-optionallong